docs/releases/v2023.1.1.rst

```diff
@@ -0,0 +1,49 @@
+Gluon 2023.1.1
+==============
+
+Important notes
+---------------
+
+Upgrades to v2023.1.1 and later releases are only supported from releases v2021.1 and later.
+This is due to migrations that have been removed to simplify maintenance.
+
+
+Bugfixes
+--------
+
+- x86: fix config loss during direct upgrades from v2021.1.x to v2023.1.x  (`#2972 <https://github.com/freifunk-gluon/gluon/pull/2972>`_)
+
+- tunneldigger: remove watchdog script that always failed and caused endless restarts (`#2987 <https://github.com/freifunk-gluon/gluon/pull/2987>`_)
+
+- gluon-wan-dnsmasq: fix restart handling and disable DNS caching on the WAN side (`#2991 <https://github.com/freifunk-gluon/gluon/pull/2991>`_)
+
+- fix upgrades of the D-Link DIR-860L B1 (`#2990 <https://github.com/freifunk-gluon/gluon/pull/2990>`_)
+
+- fix build issues with groff 1.23.0+ on the build host (`#3001 <https://github.com/freifunk-gluon/gluon/pull/3001>`_)
```

Comment:
```suggestion
- Fix build issues with groff 1.23.0+ on the build host (`#3001 <https://github.com/freifunk-gluon/gluon/pull/3001>`_)
```
We usually capitalize all items in such lists (except for the package name prefixes)